### DISCUSSION TITLE
is a bug? wifi setting? xiao_c3 1.9.0.45

### DISCUSSION THREAD

**Discussion by gwnpeter** [nanoframework/Home#1450] [Q&A]
![379a4007ef3f41e79671ad072b18413](https://github.com/nanoframework/Home/assets/162097756/f4b1f4ff-6557-423a-aeaf-3dd4449affb2)
![b41225d8afb154f672cfa064d878229](https://github.com/nanoframework/Home/assets/162097756/df2294ae-005d-4223-8076-3ce1c0e8622d)

**Comment by Ellerbach** [comment]
please use --masserase when flashing your device.

**Comment by gwnpeter** [comment]
Goooooooooooooooooood
